docs/dynamo.md

```diff
@@ -86,5 +86,15 @@ average | 1.0779
 
 We are currently working on the optimizer support and that will be availiable on nightly soon but won't be in the 2.0 release.
 
+### Feature gaps
+There are few gaps we want to call out that are preventing us from using the TorchDynamo on larger scale models.
+
+1. TorchDynamo does not trace the communication ops(like `all_reduce` and `all_gather`) yet. It will let the communication ops run on eager mode, which is not ideal for PyTorch/XLA use case since we would want the compiler to see all communication ops in the same graph.
+2. TorchDynamo will trace forward and backward into separate graphs. For PyTorch/XLA it is important to let the XLA compiler see the whole step as one graph to best optimize the speed. There is also a fixed overhead to launch every device execution which make executing multiple graphs per training step less ideal.
```

Comment:
2. TorchDynamo will trace forward, backward, and optimizer into three separate graphs. For PyTorch/XLA it is important to let the XLA compiler see all ops in one graph to best optimize for performance. There is also a fixed overhead to launch every device execution which make executing multiple graphs (per training step) less ideal.